gateway/src/apicast/policy/rate_limit_headers/cache_entry.lua

```diff
@@ -41,4 +42,25 @@ function _M:reset(max, remaining, reset)
   self.reset = countdown_counter.new(reset, now())
 end
 
+function _M:export()
+  return string.format("%s#%s#%s",
+    self.limit:__tostring(),
+    self.remaining:__tostring(),
+    self.reset:remaining_secs_positive(now()))
+end
+
+function _M.import(usage, exported_data)
+  if not usage then
+    return nil
+  end
+
+  if not exported_data then
+    return _M.Init_empty(usage)
+  end
+
+  local data = stringx.split(exported_data, "#")
```

Comment:
Nitpick: the "#" could be extracted to a constant that could be used in this method and in `export()`.